.github/workflows/docs.yml

```diff
@@ -27,10 +27,5 @@ jobs:
       with:
         python-version: '3.9'
 
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip
-        python -m pip install --upgrade nox
-
     - name: Build documentation
-      run: python -m nox -s docs
+      run: pipx run nox --error-on-missing-interpreters -s docs
```

Comment:
Where is pipx getting installed? Is the default runner including pipx by default?

Reply:
yes, all GHA runners come with pipx pre installed.